packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -2076,15 +2076,18 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   parseMeetingInfo(meetingInfo) {
+    const webexMeetingInfo = meetingInfo.body;
+
     // MeetingInfo will be undefined for 1:1 calls
-    if (meetingInfo && meetingInfo.body && !(meetingInfo.errors && meetingInfo.errors.length > 0)) {
-      this.conversationUrl = meetingInfo.body.conversationUrl || this.conversationUrl;
-      this.locusUrl = meetingInfo.body.locusUrl || this.locusUrl;
-      this.setSipUri(this.config.experimental.enableUnifiedMeetings ? meetingInfo.body.sipUrl : meetingInfo.body.sipMeetingUri || this.sipUri);
+    if (webexMeetingInfo && !(meetingInfo.errors && meetingInfo.errors.length > 0)) {
+      this.conversationUrl = webexMeetingInfo.conversationUrl || this.conversationUrl;
+      this.locusUrl = webexMeetingInfo.locusUrl || this.locusUrl;
+      this.setSipUri(this.config.experimental.enableUnifiedMeetings ? webexMeetingInfo.sipUrl : webexMeetingInfo.sipMeetingUri || this.sipUri);
       if (this.config.experimental.enableUnifiedMeetings) {
-        this.meetingNumber = meetingInfo.body.meetingNumber;
+        this.meetingNumber = webexMeetingInfo.meetingNumber;
       }
-      this.owner = meetingInfo.body.owner || meetingInfo.body.hostId || this.owner;
+      this.owner = webexMeetingInfo.owner || webexMeetingInfo.hostId || this.owner;
+      this.permissionToken = webexMeetingInfo.permissionToken;
```

Comment:
the wbxappapi spec at https://wiki.cisco.com/display/HFWEB/MeetingInfo+api also has a permissionTokenTtl in the response. I'm not sure what the client is supposed to do with it, should it be checked before calling Locus?

Reply:
I did not see anything mentioned in the document , Can check with the server team about it